preview_generator/preview/builder/image__wand.py

```diff
@@ -145,7 +145,7 @@ def image_to_jpeg_wand(
                 dims_out=preview_dims
             )
             image.resize(resize_dim.width, resize_dim.height)
-
+            image.strip()
```

Comment:
@jumenzel can you add this comment over the strip() : 
# INFO - jumenzel - 2019-03-12 - remove metadata, colors-profiles of this image.